cellfinder_core/classify/classify.py

```diff
@@ -23,13 +24,24 @@ def main(
     model_weights,
     network_depth,
     max_workers=3,
+    *,
```

Comment:
I have never come across this syntax before! For anyone else https://stackoverflow.com/questions/14301967/bare-asterisk-in-function-arguments

Reply:
I've come to view it as best practice when using keyword arguments, as it forces users to write code that is more readable by forcing them to write out argument names.